xml/System.Configuration.Install/AssemblyInstaller.xml

```diff
@@ -124,7 +123,6 @@
         <Parameter Name="commandLine" Type="System.String[]" Index="1" />
       </Parameters>
       <Docs>
-        <param name="filename">The file name of the assembly to install.</param>
```

Comment:
this is still being deleted

Reply:
I believe I've found the culprit for this issue, it's in another plugin in the pipeline ... which would explain why every test I've thrown at this did not reproduce the issue locally :) I'm verifying that, so I should have a new build for you early next week 👏

Reply:
This issue will be resolved with this PR:
https://github.com/docascode/tripled/pull/1